modules/gapi/test/rmat/rmat_tests.cpp

```diff
@@ -116,11 +116,11 @@ class RMatAdapterForBackend : public RMat::Adapter {
 // we have some specific data hidden under RMat,
 // test that we can obtain it via RMat.as<T>() method
 TEST(RMat, UsageInBackend) {
-    int i = std::rand();
+    int i = 123456;
```

Comment:
123456 is definitely a random number. Is call of cv::RNG() equal to call of rand()?

Reply:
randomization is not really necessary for this test case.

/cc @rgarnov

Reply:
I'm wondering why you didn't use analogue in this case? And i don't mind 123456.

Reply:
Ok